pom.xml

```diff
@@ -27,7 +27,7 @@ limitations under the License.
   <groupId>com.linkedin.calcite</groupId>
   <artifactId>calcite</artifactId>
   <packaging>pom</packaging>
-  <version>1.21.0.130</version>
+  <version>1.21.0.133</version>
```

Comment:
Does this mean that every pom file needs to be updated whenever there is a new version published?

Reply:
No, the the mvn versions command in the [publish script](https://github.com/linkedin/linkedin-calcite/blob/li-1.21.0/.lipublish/publish.sh#L46) will do that automatically. It seems fresh checkouts were having a problem doing that because the version in the [parent pom](https://github.com/linkedin/linkedin-calcite/blob/li-1.21.0/pom.xml#L30) and the [child poms](https://github.com/linkedin/linkedin-calcite/blob/li-1.21.0/core/pom.xml#L23) didn't match -- this just makes them match. That command will keep them matched so nothing more should required.

Reply:
Thanks.